i3lock.c

```diff
@@ -1829,9 +1992,12 @@ int main(int argc, char *argv[]) {
             case 999:
                 debug_mode = true;
                 break;
+            case 998:
+                image_raw_format = strdup(optarg);
+                break;
             default:
                 errx(EXIT_FAILURE, "Syntax: i3lock [-v] [-n] [-b] [-d] [-c color] [-u] [-p win|default]"
-                                   " [-i image.png] [-t] [-e] [-f]\n"
+                                   " [-i image.png] [-t] [-e] [-f] [-l]\n"
```

Comment:
I don't see the case for `l` - I think that the tty lock switching isn't being checked.